### DISCUSSION TITLE
Record runs to the dashboard

### DISCUSSION THREAD

**Discussion by ernstjonny** [cypress-io/cypress#16185] [Cypress Cloud]
I am new to Cypress. I would like to see my runs in the dashboard. I use the command:
"npm run cy:open --env=prod" to run Cypress. How can I add the "--record --key 2f1b123c-3257-41b7-aaca-dc322227fcc4" command to see the runs in the dashboard?
I am getting "cypress open --env environment=%npm_config_env%" in the CMD when I run the command as is.

**Comment by ernstjonny** [comment]
I found out. Thanks.

**Reply by igor98bn** [reply]
I was just about to answer
